<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="btVotar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(52, 101, 164);</string>
   </property>
   <property name="text">
    <string>VOTAR</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtVoto">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbAguardando">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Aguardando Liberação</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
